.github/workflows/image-centos.yml

```diff
@@ -26,7 +30,13 @@ jobs:
           - cloud
         architecture:
           - amd64
-          # - arm64
+          - arm64
+        build_arm64:
+          - ${{ inputs.build-arm64 == true || github.event.schedule == '0 0 * * 1' }}
+        exclude:
+          - {architecture: amd64, build_arm64: true}
```

Comment:
This exclude mechanism forcing a double negation is annoying but I don't see how to simplify it.

Reply:
Yes I agree, but it is for the same reason as described in https://github.com/canonical/lxd-ci/pull/154#discussion_r1572492494.